src/status/StatusConnected.js

```diff
@@ -3,8 +3,22 @@ import { withTranslation, Trans } from 'react-i18next'
 import { connect } from 'redux-bundler-react'
 import { humanSize } from '../lib/files'
 
-export const StatusConnected = ({ t, peersCount, repoSize }) => {
+export const StatusConnected = ({ t, peersCount, repoSize, downloadedSize, sharedSize }) => {
   const humanRepoSize = humanSize(repoSize || 0)
+  downloadedSize = downloadedSize || 0
```

Comment:
nit: we should prefer the null coalescence operator (`??`) instead of the OR operator (`||`) for default values if possible. I assume we also want to make sure downloadedSize is a number? `Number(downloadedSize ?? 0)` 

ditto for other vars here.